static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-election-interactive-end.js

```diff
@@ -6,13 +6,16 @@ define([
     'lib/config',
     'raw-loader!common/views/acquisitions-epic-control.html',
     'raw-loader!common/views/acquisitions-epic-slice.html',
+    'common/modules/commercial/acquisitions-copy',
+
 ], function (
     contributionsUtilities,
     $,
     geolocation,
     template,
     config,
-    epicControlTemplate
+    epicControlTemplate,
+    acquisitionsCopy
```

Comment:
Thanks Guy, great spot. This must have crept in during my git nightmare. I've fixed this now and will reproduce the epics to make sure nothing else has broken since I made the change